Chapter-3/dependency_parsing.py

```diff
@@ -41,7 +41,14 @@
 dep_tree = [parse.tree() for parse in result][0]
 print dep_tree
 dep_tree.draw()
-                           
+
+'''
```

Comment:
Can you uncomment this and send the pull request. If this is working, its ok to put the code snippet directly in the main file. No need to put it in quotes.